### PR TITLE
Add dropdown for basic ammo types

### PR DESCRIPTION
**Pull request by njfox**
This adds a dropdown to the item sheet to allow the user to specify the type of ammo that an item uses. It's only cosmetic now but ideally it would be tied to inventory later on for automated ammo tracking.

Only basic ammo types are implemented because I wasn't sure the best way to handle special ammo types, as they are supposed to have some restrictions (e.g. Expansive Ammunition is only allowed for Arrows, Bullets and Slugs) and I wasn't sure of the best way to validate that. Do you have any suggestions @Gerdofal ?

This is to address #38

### CODE REVIEW

File: templates/item/item-sheet.html [hunk 1]
```diff
@@ -45,6 +45,20 @@ <h1 class="charname">
...
+          <option value="none">{{localize "CPRED.none"}}</option>
+          <option value="mpistol">{{localize "CPRED.mpistol"}}</option>
+          <option value="hpistol">{{localize "CPRED.hpistol"}}</option>
+          <option value="vhpistol">{{localize "CPRED.vhpistol"}}</option>
+          <option value="slug">{{localize "CPRED.slug"}}</option>
+          <option value="rifle">{{localize "CPRED.rifle"}}</option>
+          <option value="arrow">{{localize "CPRED.arrow"}}</option>
+          <option value="grenade">{{localize "CPRED.grenade"}}</option>
+          <option value="rocket">{{localize "CPRED.rocket"}}</option>
+        {{/select}}
```
Comment (Gerdofal) [inline templates/item/item-sheet.html]: Maybe we can make this a list, instead of hard coding it? Easier to modify in the future and re-use if we do that.

Comment (Gerdofal) [reply templates/item/item-sheet.html]: Actually. I take that back. That's beyond the scope of the next update. Just hard code it for now. In an update or two I'm going to be making all such things lists and we will do that then.

**Review by Gerdofal — APPROVED** [review]

**Review by Gerdofal — APPROVED** [review]

**Comment by Gerdofal** [comment]
The only other thing needed is a minor change to template.json to make it save

**Comment by njfox** [comment]
Would that look like this?

```
        "ammotype": {
          "value": "none"
        },
```

File: templates/item/item-sheet.html (hunk 1 above)
Comment (njfox) [reply templates/item/item-sheet.html]: Will lists make it easier to validate the special ammo types or would that have to wait for the data redesign/rework?

**Comment by Gerdofal** [comment]
Close. Have to setup telco
 For a new employee. Will reply more later.

**Comment by Gerdofal** [comment]
It would actually be a new property of the "weapon" template which starts at line 940 of template.json

> 
> "weapon": {
>         "damage": {
>           "value": "3d6"
>         },
>         "ammo": {
>           "min": 0,
>           "max": 16,
>           "value": 0
>         },
>         "weapontype": {
>           "value": "pistol"
>         },
>         "burst": {
>           "value": false
>         },
>         "fullauto": {
>           "value": false
>         },
>         "skill": {
>           "value": "marksmanship"
>         },
>         "rof": {
>           "value": 1
>         }
>       },
>

**Comment by njfox** [comment]
Gotcha, that's where I was planning to put that snippet, right below the ammo property. Just wasn't sure if I needed to add anything else

**Review by Gerdofal — APPROVED** [review]

**Pull request merged**